### DISCUSSION TITLE
touch a file when a cert has been updated from cron job

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [acmesh-official/acme.sh#4834] [General]
I was wondering if I run a cron like this

`runuser -u acme -- /home/acme/acme.sh --cron --home "/home/acme" >/dev/null`

how I can touch a file somewhere (another cronjob checks if this file is touched and reloads the configs.

Does the exit status reveal if a cert was updated?

**Comment by neilpang** [comment]
use `reloadcmd`:  https://github.com/acmesh-official/acme.sh#3-install-the-cert-to-apachenginx-etc

Or, use hooks: `https://github.com/acmesh-official/acme.sh/wiki/Using-pre-hook-post-hook-renew-hook-reloadcmd`

**Reply by f1-outsourcing** [reply]
Shit I already issued some without the --renew-hook / --post-hook. How can I add renew-hooks to existing certs?

**Reply by f1-outsourcing** [reply]
oh I already found Le_RenewHook=''
